client/store/orders.js

```diff
@@ -0,0 +1,61 @@
+import Orders from '~/api/orders'
+export const state = () => ({
+  orders: [],
+  isOpen: false,
+  timeBeforeDelete: {
+    minutes: 14,
```

Comment:
https://stackoverflow.com/questions/65637458/how-to-make-a-simple-countdown-in-vue3